kolibri/plugins/style_guide/assets/src/app.js

```diff
@@ -3,6 +3,9 @@ const Vue = require('kolibri.lib.vue');
 const RootVue = require('./views');
 const router = require('kolibri.coreVue.router');
 const { navigationMenuRoutes } = require('./views/shell/navigation-menu');
+const Vuep = require('./third_party/vuep/vuep');
```

Comment:
Note - we should add these dependencies only in the style_guide plugin (so do `yarn add vuep codemirror` from the `style_guide` directory).